docs/getting_started/requirements_and_system_configuration.md

```diff
@@ -93,14 +93,13 @@ eZ software is developed to work with *any* web browser that support modern sta
 
 These setups have been undergone some additional manual testing and is known to work.
 
--   Mozilla® Firefox® most recent stable version (tested\\ on\\ Firefox\\ 50)
--   Google Chrome™ most recent stable version (tested\\ on\\ Chrome\\ 55) 
--   Microsoft® Edge® most recent stable version (tested\\ on Edge\\ 38) 
+-   Mozilla® Firefox® most recent stable version *(tested on Firefox 54)*
```

Comment:
TBH you can omit the version number here, it's implied in _latests version_, and having it here just means it is almost always out of date :)